### ISSUE TITLE
Error when running

### ISSUE THREAD

**Issue by edugodoyo** [theohbrothers/ConvertOneNote2MarkDown#114]
## Bug

Error:

Convert-OneNote2MarkDown : Retrieving the COM class factory for component with CLSID
{DC67E480-C3CB-49F8-8232-60B0C2056C8E} failed due to the following error: 80080005 Server execution failed (Exception
from HRESULT: 0x80080005 (CO_E_SERVER_EXEC_FAILURE)).
At C:\Users\edugoliv\Downloads\ConvertOneNote2MarkDown-v2.ps1:1423 char:5
+     Convert-OneNote2MarkDown @params
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Convert-OneNote2MarkDown

Convert-OneNote2MarkDown : at New-OneNoteConnection, C:\Users\edugoliv\Downloads\ConvertOneNote2MarkDown-v2.ps1: line
530
at Convert-OneNote2MarkDown, C:\Users\edugoliv\Downloads\ConvertOneNote2MarkDown-v2.ps1: line 1359
at <ScriptBlock>, C:\Users\edugoliv\Downloads\ConvertOneNote2MarkDown-v2.ps1: line 1423
at <ScriptBlock>, <No file>: line 1
At C:\Users\edugoliv\Downloads\ConvertOneNote2MarkDown-v2.ps1:1423 char:5
+     Convert-OneNote2MarkDown @params
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Convert-OneNote2MarkDown

## Expectation

No error.

## Discussion

After running for a minute.

## Environment

Windows 10 Version = 20H2 Build 19042.1288
OneNote version = 1808 (Msft office Professional Plus 2019)
Pandoc Version = 2.16.2

**Output of Configuration**

For example:

Configuration:
dryRun: 1
notesdestpath: C:\Users\edugoliv\Downloads\onmd
targetNotebook:
usedocx: 1
keepdocx: 2
docxNamingConvention: 1
prefixFolders: 1
mdFileNameAndFolderNameMaxLength: 32
medialocation: 1
conversion: markdown_mmd-simple_tables-multiline_tables-grid_tables+pipe_tables
headerTimestampEnabled: 1
keepspaces: 1
keepescape: 1
newlineCharacter: 1
exportPdf: 1

**Output of `$PSVersionTable`**

For example:

Name                           Value
----                           -----
PSVersion                      5.1.19041.1237
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1237
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

**Comment by SlevinBE** [comment]
Hi, you might be using the Windows Store version of Onenote. If you open OneNote and see "OneNote for Windows 10" in the page title, that's probably the case and what's causing this error. According to the requirements mentioned in the readme, you need to use the other version of OneNote: https://github.com/theohbrothers/ConvertOneNote2MarkDown#requirements

**Comment by AndersGad** [comment]
I had the same buck and fixed it, by running vs code as an administrator and using the newest powershell extension for vs code.

**Comment by PyPS-PowerUser** [comment]
I have the same issue, first tried it on my machine then in a new virtual machine with win 10. With the OneNote for Windows 10 at least it works in DryRun (but not the actual one) but in the Office OneNote it comes up with the same error.

Using MS OneNote for M365 MSO (Version 2201)

**Comment by PyPS-PowerUser** [comment]
So I tried a lot of different things about the issue and found a way to resolve it.
tl;dr: I ran PowerShell as Admin but not OneNote, reopened OneNote as Admin and it worked, somewhat.

So what I did that worked:

1. Create another windows 10 virtual machine
2. Install Office 2016 (had an old CD + Key flying around)
3. Install chocolate and padoc as instructed
4. Run PowerShell and OneNote as Administrator

Caveat: 

After finding this out I tried it on my normal machine but there I get an error that the file can't be opened because OneNote is run in administrative mode.

Here some links and possible other avenues of help:
https://github.com/SjoerdV/ConvertOneNote2MarkDown/issues/11 <- Here I read about the admin issue
https://stackoverflow.com/questions/45084748/exception-calling-ctor-with-0-arguments-retrieving-the-com-class-factory-for-c
https://docs.microsoft.com/en-us/sysinternals/downloads/procmon
https://docs.microsoft.com/en-us/troubleshoot/windows-server/application-management/error-8008005-when-start-complus-applications

Only the first one helped in the end and was the last fix applied.

**Comment by leojonathanoh** [comment]
The cause is explained here: https://github.com/theohbrothers/ConvertOneNote2MarkDown/issues/102#issuecomment-906783027

Since this is a very common error, I'll open a PR to add docs for this error.
